src/pages/Settings/Settings.tsx

```diff
@@ -0,0 +1,39 @@
+import styled from 'styled-components';
+import DoNotShowPositionReset from '../../components/Global/SettingsComponents/DoNotShowPositionReset';
+import { Text } from '../../styled/Common';
+
+const Container = styled.div`
+    width: 100%;
+    margin: 0 auto;
+    padding: 20px;
+    border-radius: 10px;
+    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
```

Comment:
Would consider if these unique stylings are necessary.  Could we just use the default 'rounded' style and a more standard box shadow vs defining new ones here.